reference-architecture/vmware-ansible/roles/docker-storage-setup/tasks/main.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: create the docker-storage-setup config file
+  copy:
+     src: docker-storage-setup
+     dest: /etc/sysconfig/docker-storage-setup
+     owner: root
+     group: root
+     mode: 0644
+- name: start docker
+  service:
+      name: docker
+      state: started
+      enabled: true
```

Comment:
I wouldn't expect a docker-storage-setup role to start the docker daemon. If the goal is to trigger the storage setup, then you could use a handler to run the docker-storage-setup service on config file change.